http_check/datadog_checks/http_check/data/conf.yaml.example

```diff
@@ -2,243 +2,243 @@ init_config:
   ## @param ca_certs - string - optional
   ## Change default path of trusted certificates
   #
-  ## ca_certs: /etc/ssl/certs/ca-certificates.crt
+  # ca_certs: /etc/ssl/certs/ca-certificates.crt
```

Comment:
Should we have a placeholder here?

Reply:
it should in theory.. but http check is special in a sense that it's used heavily as a getting started on integrations... we could change it if you want.

Reply:
This is a valid default value, so I think we should keep it.